source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md

```diff
@@ -65,3 +65,11 @@ You can destroy the tunnel by using the port value found within the **Connection
 ```bash
 ps -fU USERNAME | grep "ssh -f" | grep "PORT:" | awk '{print $2}' | xargs kill
 ```
+
+## Troubleshooting
+### Lost Connection to MySQL Server

```

Comment:
@nataliejeremy I see what you mean. It's nice to have the error as the header, but I don't know if this is the only error returned for an idle db server - there may be more depending on the service making the request.

@CSE can you weigh in? ^